float64 input_temp
---
float64 temp_diff
